`make full_test` fails on a clean Chronos checkout

```
187 10:02:37  chronos  [dev]$ make full_test
clang++-3.6   -MMD -MP -O0 -ggdb3 -std=c++11 -Wall -Werror -DUNIT_TEST -fprofile-arcs -ftest-coverage -fno-access-control -Imodules/gmock/gtest/include -Imodules/gmock/include -I../modules/cpp-common/test_utils -Isrc/include -Imodules/cpp-common/include -Imodules/rapidjson/include -Imodules/sas-client/include -Ibuild/usr/include -Imodules/cpp-common/test_utils -Isrc/test -c src/main/chronos_internal_connection.cpp -o build/chronos_test/chronos_internal_connection.o
In file included from src/main/chronos_internal_connection.cpp:45:
modules/sas-client/include/sas.h:42:10: fatal error: 'config.h' file not found
#include <config.h>
         ^
1 error generated.
make: *** [build/chronos_test/chronos_internal_connection.o] Error 1
```

--- a/doc/Development.md
+++ b/doc/Development.md
@@ -37,10 +37,17 @@ This accesses the repository over SSH on Github, and will not work unless you ha
     git config --global url."https://github.com/".insteadOf git@github.com:
     git clone --recursive git@github.com:Metaswitch/chronos.git
 
+## Building Dependencies
+
+To build Chronos's 3rd party dependencies, change to the top-level `chronos`
+directory and issue `make submodules`.  You will want to re-run this command
+after changing any submodule or pulling a newer release of Chronos that
+updates the submodule references (after running `git submodule update`).
+
 ## Building Binaries
 
-To build Chronos and all its dependencies, change to the top-level `chronos`
-directory and issue `make`.  
+To build Chronos, change to the top-level `chronos` directory and issue `make`.
+You must have built the dependencies first (see above).
 
 On completion,
 
@@ -50,7 +57,8 @@ On completion,
 ## Building Debian Packages
 
 To build Debian packages, run `make deb`.  On completion, Debian packages
-are in the parent of the top-level `chronos` directory.
+are in the parent of the top-level `chronos` directory.  You must have built
+the dependencies first (see above).
 
 `make deb` can push the resulting binaries to a Debian
 repository server.  To push to a repository server on the build machine, set

